CHANGELOG.rst

```diff
@@ -233,7 +233,11 @@ time or change existing behaviors in order to make them less surprising/more use
   Thanks `@Vogtinator`_ for reporting and `@RedBeardCode`_ and
   `@tomviner`_ for the PR.
 
-*
+* Added confcutdir in testing/test_conftest.py::test_conftest_import_order and
```

Comment:
I wanted to say that I didn't know about your suggestion. Should i delete the entry?